lib/tasks/plantain.rake

```diff
@@ -66,18 +74,14 @@ namespace :plantain do
   desc "Seed fixture data to Solr"
   task :seed do
     puts "Seeding index with data from spec/fixtures/ead..."
-    Dir.glob("spec/fixtures/ead/*.xml").each do |file|
-      system("FILE=#{file} rake arclight:index") # no REPOSITORY_ID
-    end
 
-    Dir.glob("spec/fixtures/ead/*").each do |dir|
-      next unless File.directory?(dir)
+    Dir.glob("eads/**/*.xml").each do |file|
```

Comment:
This task used to seed fixture data. Did we want it to index all eads instead? Do we need 2 tasks?

Reply:
I can separate this into two Rake tasks.